torch/csrc/jit/python/script_init.cpp

```diff
@@ -710,6 +710,22 @@ void extra_files_to_python(const ExtraFilesMap& m, const py::dict& pydict) {
   }
 }
 
+void compilationUnit_py_define(
+    CompilationUnit& cu,
+    const std::string& src,
+    const ResolutionCallback* rcb,
+    const uint32_t _frames_up) {
+  if (rcb && *rcb) {
+    cu.define(c10::nullopt, src, pythonResolver(*rcb), nullptr);
+  } else {
+    py::object py_default_rcb =
+        py::module_::import("torch._jit_internal")
```

Comment:
One more thing - why is this `module_` instead of `module`?

Reply:
Sleeping on the `_`....

Reply:
Thank you for catching all these!